src/main/java/invtweaks/InvTweaksHandlerShortcuts.java

```diff
@@ -437,16 +437,18 @@ private void moveAll(ShortcutConfig shortcut, ItemStack stackToMatch) {
 
                     // Continue if movement succeeded, there is another slot to try, or we're dropping items.
                     // In reverse: fail if movement failed, AND there are no other slots AND we're not dropping.
-                    if(success || (newIndex != toIndex) || (shortcut.action == ShortcutSpecification.Action.DROP)) {
+                    if (newIndex == toIndex) {
```

Comment:
This will break drops, won't it? The `toIndex` and `newIndex` for a multi-item drop is always the same (DROP_SLOT, -999)